VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -454,8 +461,10 @@ public void testUnitFragmentCreation() {
     // TODO: Robolectric doesn't have a Gradle-based parameterized test suite, need to implement it.
     @Ignore
     @RunWith(Parameterized.class)
-    public static class ParameterizedFragmentCreation {
-        @Parameterized.Parameters(name= "{index}: testUnitFragmentCreation({0})={1}")
+    public static class ParameterizedFragmentCreation extends UiTest {
```

Comment:
You can basically copy over the javadoc from the `testUnitFragmentCreation()` test, changing it as appropriate to describe the parameterization done here.